### PR TITLE
Parameterize networkid for geth

### PR DESCRIPTION
**Pull request by pgebheim**

### CODE REVIEW

File: geth-poa/start.sh [hunk 1]
```diff
@@ -32,11 +32,19 @@ setup_chain_dir() {
...
+      geth --datadir "${ROOT}/chain-template" --keystore "${ROOT}/keys" init "${ROOT}/genesis.json"
+
+      echo ${NETWORK_ID:-12346} > "${ROOT}/networkid"
+    fi
+
     echo "${ROOT}/chain not mounted, transactions will be ephemeral"
     mv ${ROOT}/chain-template ${ROOT}/chain
   else
     # Chain dir exists
-    if [ -d /geth/chain/geth/chaindata ]; then
```
Comment (epheph) [inline geth-poa/start.sh:39]: thx for fixing this

**Review by epheph — APPROVED** [review]
🐐

File: geth-poa/start.sh (hunk 1 above)
Comment (pgebheim) [reply geth-poa/start.sh:39]: at your service

File: geth-poa/start.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ source ./common_start.sh
 node_start() {
   # geth is dumb and won't let us run it in the background, and nohup redirects to file when run in a script
   nohup geth \
-    --networkid 12346 \
+    --networkid "$(cat "$ROOT/networkid")" \
     --datadir "${ROOT}/chain" \
```
Comment (bthaile) [inline geth-poa/start.sh:16]: so networked is a file with network id we want? so we could have many network ids, dev-node-geth and dev-pop-geth could have their own?

**Pull request merged**